audio/out/ao_openal.c

```diff
@@ -59,23 +59,25 @@
 #define MAX_CHANS MP_NUM_CHANNELS
 #define NUM_BUF 128
 #define CHUNK_SAMPLES 256
-static ALuint buffers[MAX_CHANS][NUM_BUF];
-static ALuint sources[MAX_CHANS];
+static ALuint buffers[NUM_BUF];
+static ALuint source;
 
-static int cur_buf[MAX_CHANS];
-static int unqueue_buf[MAX_CHANS];
+static int cur_buf;
+static int unqueue_buf;
 
 static struct ao *ao_data;
 
 struct priv {
     ALenum al_format;
     int chunk_size;
+    int direct_channels;
 };
 
 static void reset(struct ao *ao);
 
 static int control(struct ao *ao, enum aocontrol cmd, void *arg)
 {
+    ALboolean mute;
```

Comment:
1) can be declared inside `case:`
2) the argument passed is a `bool`, which is not guranteed to be identical to `ALboolean`

Reply:
Done.